salt/metalk8s/service-configuration/deployed/init.sls

```diff
@@ -0,0 +1,10 @@
+#
+# States to deploy cluster and service configurations
+#
+
+# Create the Namespace before deploying ConfiMap objects
+# If we include service configurations in their respective addons deploy states,
+# then salt will fail while rendering the jinja since the Namespace object does
```

Comment:
I think you should say that is that '"ConfigMap' does not exist yet" even if the first error you get is because the namespace do not exist the issue here is that the CM do not exist (even if you create the namespace you will get an error because the ConfigMap do not exist)